src/services/Parser/ParserAbstract.ts

```diff
@@ -1,6 +1,6 @@
-import * as fs from "fs";
+import fs from "fs";
 import { ParsedData, MetaData, Content } from "./Types";
-import {LOCAL_FILES_PATHS} from "../../../utils/constants.js"
+import {LOCAL_TMP_PATHS} from "../../../utils/constants.js"
```

Comment:
C'est du détail, mais les espaces sont bien, comme tu as fait ligne 2 :
```suggestion
import { LOCAL_TMP_PATHS } from "../../../utils/constants.js"
```

Reply:
d'accord, je modifie ça dans la branche parser